metricbeat/magefile.go

```diff
@@ -23,15 +23,14 @@ import (
 	"context"
 	"fmt"
 	"os"
-	"regexp"
 	"strconv"
 	"time"
 
 	"github.com/magefile/mage/mg"
 	"github.com/magefile/mage/sh"
-	"github.com/pkg/errors"
 
 	"github.com/elastic/beats/dev-tools/mage"
+	metricbeat "github.com/elastic/beats/metricbeat/scripts/mage"
```

Comment:
This is not a Metricbeat, don't call it like this. `mageMetricbeat` is more correct.

Reply:
Yeah, I am not a superfan of this name. I took it from filebeat, where the equivalent package is imported as `filebeat`. wdyt about leaving it as is by now for coherence with other beats?

Reply:
maybe change to mageMB and mageFB later 😬

Reply:
Oook, I'll follow with the family of `mbpackages` :slightly_smiling_face:

Reply:
Well though, lets keep current naming and leave this discussion for another PR to don't block this, there have been some recent changes in naming recently (https://github.com/elastic/beats/pull/12314).